Karl/karl.h

```diff
@@ -313,17 +463,17 @@ class GenericKarl final {
             }
           }();
 
+          const auto now = current::time::Now();
```

Comment:
More accurate time skew calculation :)
(I also used it in some intermediate version to update status along with the `Publish` to the stream, but then changed the logic.)